src/Extension/PathfinderControllerExtension.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace CodeCraft\Pathfinder\Extension;
+
+use CodeCraft\Pathfinder\Model\Pathfinder;
+use SilverStripe\Core\Extension;
+
+/**
+ * Extend a controller to support an interactive Pathfinder
+ *
+ * @property PathfinderPageController owner
+ */
+class PathfinderControllerExtension extends Extension
```

Comment:
```suggestion
class PathfinderPageControllerExtension extends Extension
```

If refactor recommendation is considered above then might need to update this

Reply:
Sweet, good suggestion, but not quite in line with the strategy I had in mind